tests/cypress/e2e/features/skeletons_pipeline.js

```diff
@@ -223,5 +223,18 @@ context('Manipulations with skeletons', { scrollBehavior: false }, () => {
 
             cy.removeAnnotations();
         });
+
+        it('Copy/paste a skeleton shape', () => {
+            cy.on('window:console:error', (error) => {
+                throw new Error(`Console error detected: ${error}`);
+            });
```

Comment:
I dont think we need to check console. When Im trying to reproduce the issue before the fix, UI crushes. The correct copying should be enough
![image](https://github.com/cvat-ai/cvat/assets/50956430/e16584c3-d8e8-4187-9420-07f24efb0d0b)